src/pm-conversations/__tests__/pmConversationsModel-test.js

```diff
@@ -38,7 +38,11 @@ describe('reducer', () => {
         { user_ids: [2, 1].map(makeUserId), max_message_id: 345 }, // user_ids out of order
       ];
       const expected = {
-        map: Immutable.Map([['', 234], ['1', 123], ['1,2', 345]]),
+        map: Immutable.Map([
+          ['', 234],
+          ['1', 123],
+          ['1,2', 345],
+        ]),
```

Comment:
(For when we do the Prettier upgrade, whether now or later:)

If you feel like just adding some prettier-ignore comments on this kind of diff hunk, to keep these blobs of test data compact, I'd be for it.